frontend/assets/style/components/_forms.scss

```diff
@@ -300,29 +336,76 @@ p.error {
       margin-right: $spacer-sm;
       border: 1px solid $general_0;
       transform: translateY(2px);
+      transition: box-shadow 150ms, background-color 150ms;
     }
 
     &::after {
       position: absolute;
       width: 6px;
       height: 6px;
-      background-color: $primary_0;
+      background-color: $background_0;
       top: 11px;
```

Comment:
Do we keep pixels here?

Reply:
I didn't convert these to `rem` because all form stuff is in pixels. So maybe we could create a task to convert all form stuff to rem in one go. What do u think?

Reply:
We could. I wonder if it will look good if replace 11px by 0,6875 however...